protoc-gen-twirp_php/templates/service/AbstractClient.php

```diff
@@ -245,4 +256,13 @@ protected function urlBase(string $addr): string
 
         return $addr;
     }
+
+    /**
+     * Set url path prefix. Default is twirp.
+     * @param string $prefix 
+     */
+    public function setPrefix(string $prefix)
```

Comment:
I'd slightly prefer making this a constructor parameter instead.

Reply:
I have no strong feelings either way - so I can change it to a parameter.